contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,462 @@
+client.identify({
+    distinctId: 'user_A',
+    properties: {
+        timezone: 'GMT+8',
+    },
+})
+
+/* User A will now have properties:
+{ 
+  name: 'User A',
+  location: 'London', 
+  phone: '0800-POSTHOG', 
+  timezone: 'GMT+8' 
+}
+*/
+```
+
+### How to handle duplicates of the same user
+
+It may happen that, due to implementation issues, the same user in your product has multiple users in PostHog associated with them. In these cases, you can use `$merge_dangerously` to merge multiple PostHog users into a single user.
+   
+> **Warning:** Merging users with `merge_dangerously` is irreversible and has no safeguards! Be careful not to merge users who should not be merged together.
+> 
+> Due to the dangers, we don't recommend you merge users frequently, but rather as a one-off for recovering from implementation problems.
+
+Merging users is done by sending a `$merge_dangerously` event:
+
+<MultiLanguage selector="tabs">
+
+```node
+client.capture({
+    distinctId: 'distinct_id_of_user_to_merge_into',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_to_be_merged',
+    },
+})
+```
+
+```py
+posthog.capture('distinct_id_of_user_to_merge_into', 
+  '$merge_dangerously', 
+  {'alias': 'distinct_id_of_user_to_be_merged'})
+```
+
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "distinct_id_of_user_to_merge_into",
+  Event:      "$merge_dangerously",
+  Properties: posthog.NewProperties().
+    Set("alias", "distinct_id_of_user_to_be_merged"),
+})
+```
+
+```ruby
+posthog.capture({
+    distinct_id: 'distinct_id_of_user_to_merge_into',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_to_be_merged'
+    }
+})
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+      "alias": "distinct_id_of_user_to_be_merged"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "distinct_id": "distinct_id_of_user_to_merge_into",
+    "event": "$merge_dangerously"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+#### How are properties managed when merging users?
+
+When a `User B` is merged into another `User A`, all the properties of the `User B` are added to `User A`. If there is a conflict, the properties of `User A` are prioritized over `User B`. For example:
+
+```node
+/* Assume User A has the properties:
+{ 
+        name: 'User A',
+        location: 'London',
+}
+*/
+
+/* Assume User B has the properties:
+{ 
+        name: 'User B',
+        location: 'Rome',
+        phone: '0800-POSTHOG',
+}
+*/
+
+client.capture({
+    distinctId: 'distinct_id_of_user_A',
+    event: '$merge_dangerously',
+    properties: {
+        alias: 'distinct_id_of_user_B',
+    },
+})
+
+/* User B has merged into User A. The resulting user will now have properties:
+{ 
+  name: 'User A',
+  location: 'London', 
+  phone: '0800-POSTHOG', 
+}
+*/
+```
```

Comment:
This example has been updated

Reply:
that's fine though I'm not sure it provides good incentives to merge users with different names